WcaOnRails/db/migrate/20220630233246_add_use_wca_live_for_scoretaking.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUseWcaLiveForScoretaking < ActiveRecord::Migration[6.0]
+  def change
+    add_column :competitions, :use_wca_live_for_scoretaking, :boolean, null: true, default: false
```

Comment:
Why are you allowing `null` in a boolean column?

Reply:
So that the previous competitions can be filled with `null`. Filling them with `true` didn't look good as all competitions haven't used WCA Live, and filling them with `false` will give a wrong meaning that the competition didn't use WCA Live, even though some had.

Reply:
Okay, that's a great point actually that I hadn't considered myself. Thanks for thinking about these edge cases!

Reply:
(you are free to mark this as "Resolved" at your own discretion. I simply wanted to leave this comment box open for reference because I'm mentioning it in another review comment above.)